discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -299,7 +307,7 @@ def download_installation_logs(self, path):
         self.api_client.download_cluster_logs(self.id, path)
 
     def get_install_config(self):
-        return self.api_client.get_cluster_install_config(self.id)
+        return yaml.load(self.api_client.get_cluster_install_config(self.id), Loader=yaml.SafeLoader)
```

Comment:
not sure i understand why we need it?

Reply:
I see if proxy values are set in the install config

Reply:
but you get json? why u need to change it to yaml?

Reply:
got it